src/components/Checklist/Checklist.tsx

```diff
@@ -38,7 +38,7 @@ export const Checklist: FC<ChecklistProps> = ({ activity }) => {
     addSentryBreadcrumb({
       category: BreadcrumbCategory.SUBMIT_CHECKLIST,
       data: {
-        items,
+        checklist_id: activity.id,
```

Comment:
@mohsinht this is actually not correct, we have an activity id and a checklist id

Activity id = `activity.id`
Checklist id = `activity.object.id` --> we need this one